tests/unit/multi-currency/compatibility/test-class-woocommerce-subscriptions.php

```diff
@@ -760,6 +806,52 @@ public function test_should_disable_currency_switching_return_false_when_startin
 		$this->assertFalse( $this->woocommerce_subscriptions->should_disable_currency_switching( false ) );
 	}
 
+	/**
+	 * This test is marked as skipped due to it does not currently work.
+	 *
+	 * It does not work due to WC_Payments_Explicit_Price_Formatter::should_output_explicit_price is called in the path
+	 * of maybe_get_explicit_format_for_subscription_total, and when it is called it loads the MultiCurrency instance
+	 * that is initialized before the test itself starts. The self::$multi_currency_instance->has_additional_currencies_enabled()
+	 * check returns false due to no other currencies are enabled because there's no Stripe connection.
+	 *
+	 * I could not find a way to properly mock this due to the static call to the MC instance.
+	 */
+	public function test_maybe_get_explicit_format_for_subscription_total() {
```

Comment:
I am not sure if we should leave this test in and have it skipped, or remove it. I could not find a way to mock it due to the reasons mentioned in the comment.

Reply:
We talked through this test and how to fix it in Slack: p1698840378803849-slack-C01B8KNUYSW